Drop-In/src/main/java/com/braintreepayments/api/dropin/DropInActivity.java

```diff
@@ -280,6 +282,14 @@ public void onPaymentMethodNoncesUpdated(List<PaymentMethodNonce> paymentMethodN
             if (mDropInRequest.isVaultManagerEnabled()) {
                 mVaultManagerButton.setVisibility(View.VISIBLE);
             }
+
+            for (PaymentMethodNonce nonce : paymentMethodNonces) {
+                if (nonce.getClass() == CardNonce.class) {
+                    mBraintreeFragment.sendAnalyticsEvent("vaulted-card.appear");
```

Comment:
Technically at this point we know vaulted cards exist and could be shown, but we haven't drawn a view for it. I think this might be OK though.

Reply:
Resolved in [this commit](https://github.com/braintree/braintree-android-drop-in/pull/130/commits/d5d48da09ca22f9b928193056ebd66b38328b50a).

Reply:
Yup. This is assuming nothing goes wrong with rendering the vaulted payments after we populate them.

Would you rather we implement another listener for use inside the vaulted payment methods recycler view? Likely here in the [on bind](https://github.com/braintree/braintree-android-drop-in/blob/b92db7ae2b98fd5f3afa3c716e150453167d2074/Drop-In/src/main/java/com/braintreepayments/api/dropin/adapters/VaultedPaymentMethodsAdapter.java#L38-L57).

Reply:
Nah, I think this is fine 👍

Reply:
👍